src/persistence/SubjectChangedColumnsComputer.ts

```diff
@@ -79,6 +80,11 @@ export class SubjectChangedColumnsComputer {
                     } else if (column.type === "time") {
                         normalizedValue = DateUtils.mixedDateToTimeString(entityValue);
 
+                    } else if (NumericTypes.some(t => t === column.type)) {
+                        normalizedValue = column.precision != null
+                            ? normalizedValue.toPrecision(column.precision)
+                            : normalizedValue.toString();
```

Comment:
Why do you put `.toString()` here? I guess your problem in in precision? If so, how exactly your change fixed your problem and what exact problem you had that forced you to add `toString()`? Please provide a detailed explanations on your changes. Why it didn't work before and works now? Also please add tests to your change.

Reply:
When arriving at line 84, normalizedValue is of type `number` and database value is of type `string`. To make a correct comparison, I convert the normalized value to a string by using a) `toPrecision` if the precision is known b) `toString` when the precision is not known. This is the safest comparison I could think of that wouldn't say both are equal when they are not while still recognizing equal values.

About tests, I've looked for existing tests to append but have not found any relating to the SubjectChangedColumnsComputer. Could you point me to the right files? Alternatively, I could extract the comparison to a utility function and add tests for the function specifically.

Reply:
Yeah, there are no similar tests, but we should have one...

Reply:
Also I feel like you are breaking some stuff we already have and since there are no tests in this area I feel it dangerous to merge it.

Reply:
Why I'm thinking you are breaking - because I don't understand why you need toString. You are saying that `databaseValue` is a string and `normalizedValue` is a number - are you sure its true for all number types??? Because if so, you are saying that currently check doesn't work properly for all numbers and it make redundant updates all the times, but it can't be true.

Reply:
I definitely understand your concern. 

What I aim to achieve is preventing unnecessary update queries when number columns are involved. Unless people rely on updates of unchanged numeric values it should not break anything. I could however be wrong as I don't know the intricacies of the source code. My manual tests do succeed.

Reply:
Try to remove your code and check on a simple number. What you are saying it will make unnecessary update. What I'm saying it should not do it. Can you try and check it again for a simple number case?

Reply:
I just tried with an int, it compares a number to a number there. I did my tests with decimals (Postgres).

I reckon this is because `bigNumberStrings` is turned off. So this is an edge case :thinking:

Reply:
Although, `bigNumberStrings` is only for mysql/mariadb so that's not it.

Reply:
Current tests for Postgres:

| Type             | normalizationValue | databaseValue |
|------------------|--------------------|---------------|
| decimal          | number             | string        |
| float            | number             | number        |
| numeric          | number             | string        |
| int              | number             | number        |
| double precision | number             | number        |
| real             | number             | number        |

Want me to try other types?

Reply:
I just added bigint to the table which apparently also is a string as database value

Reply:
So we need `toString()` only for `decimal`, `numeric` and `bigint`, am I right? And using it for other types would bring issues since type of normalizationValue wouldn't be equal to value emitted by `toString()`?